### ISSUE TITLE
1.1.0b2 - Jinja2 upper bound causes conflict in existing deployments

### ISSUE THREAD

**Issue by benjaoming** [readthedocs/sphinx_rtd_theme#1358]
The current pre-release will not fly with existing deployments that have already pinned the latest Jinja2 release.

Needs a decision:

* Release as is `Jinja2<3.1`
* Bump to `Jinja2<3.2` and do a new pre-release

**Comment by agjohnson** [comment]
The upper bounds were added with #1316, and replicate the fix that we instructed users to use to resolve the issue on their projects (pin `Jinja2<3.1`).

However, there is nothing on the theme that requires a specific Jinja2 version. Perhaps this is the wrong place to apply this fix entirely.

I'd instead suggest removing the dependency entirely, and fix the errors in CI by tuning dependencies with tox/etc instead.
